src/metabase/util/honeysql_extensions.clj

```diff
@@ -3,9 +3,11 @@
   (:require [clojure.string :as s]
             [honeysql
              [core :as hsql]
-             [format :as hformat]])
-  (:import honeysql.format.ToSql
-           java.util.Locale))
+             [format :as hformat]]
+            [metabase
+              [util :as u]])
```

Comment:
Instead of namespace prefixes you can do `[metabase.util :as u]` for a single namespace

Reply:
You actually don't need to import this ns at all because you don't need `u/round-to-decimals` as discussed below